pyasn1/codec/ber/decoder.py

```diff
@@ -2186,3 +2182,12 @@ def _callSubstrateFunV4asV5(substrateFunV4, asn1Object, substrate, length):
 #:     1 2 3
 #:
 decode = Decoder()
+
+def __getattr__(attr: str):
+    if attr == "tagMap":
+        warnings.warn("tagMap is deprecated. Please use TAG_MAP instead.", DeprecationWarning)
+        return TAG_MAP
+    if attr == "typeMap":
+        warnings.warn("typeMap is deprecated. Please use TYPE_MAP instead.", DeprecationWarning)
+        return TYPE_MAP
+    raise AttributeError(attr)
```

Comment:
I can probably shorten this to:
```suggestion
    deprecations = {"tagMap": "TAG_MAP", "typeMap": "TYPE_MAP"}
    if newAttr := deprecations.get(attr):
        warnings.warn(f"{attr} is deprecated. Please use {newAttr} instead.", DeprecationWarning)
        return globals()[newAttr]
    raise AttributeError(attr)
```
2 less lines, less repetitions.

Reply:
Sounds good to me!